README.md

```diff
@@ -14,6 +14,14 @@ Make sure to invoke within a Python 3.6+ virtual environment or specify Python 3
 
 Amaxa is operating system-agnostic. It has been tested primarily on Linux but is also known to work in a MINGW Windows 7 environment.
 
+Using Amaxa within Docker, execute
```

Comment:
I'm fairly new to docker and not sure what level of documentation people expect by including the docker image. The current explanation assumes some knowledge of docker. I'm happy to fill this out with a bit more detail.